src/plugins/gui/scenes/RewardsCardDashboardScene.tsx

```diff
@@ -107,18 +108,36 @@ export const RewardsCard = (props: RewardsCardProps) => {
         activeOpacity={onPress == null ? 1 : undefined}
       >
         <CardInner shouldStack={shouldStack}>
-          <Details>
+          <CardHeader>
             <VisaBrandImage source={visaBrandImage} />
-            <DetailItem>
-              <ExpiryLabel>{lstrings.rewards_card_dashboard_expires_label}</ExpiryLabel>
-              <DateLabel>{item.expirationDate.toLocaleString()}</DateLabel>
-            </DetailItem>
-          </Details>
-          {onRemovePress == null ? null : (
-            <TouchableOpacity onPress={onRemovePress}>
-              <Icon name="remove-circle-outline" size={theme.rem(1.5)} color={theme.dangerIcon} />
-            </TouchableOpacity>
-          )}
+            {onRemovePress == null ? null : (
+              <TouchableOpacity onPress={onRemovePress}>
+                <Ionicon name="remove-circle-outline" size={theme.rem(1.5)} color={theme.dangerIcon} />
+              </TouchableOpacity>
+            )}
+          </CardHeader>
+          <Space expand>
+            <Space bottom={0.5} sideways expand>
```

Comment:
Make sure all other uses of `<Space sideways>` have Jest tests that didn't change or visually check them yourself. Commit dde5bfbc6a6ddaaba5a7170c0b5cbe5645f144b4 could have caused regressions

Reply:
What's the easiest way to get snapshot testing on the loan components?